.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,41 @@
+# Issue Title
+
+Be as descriptive as you can with your title. If your issue is requesting
+a new topic, please preface the title with [NewTopic].
+
+# General
+
+We use issues to drive the discussion for changes to
+existing topics, and the creation of new topics.
+
+This way, we have those discussions before significant
+effort is spent writing a new topic. Please see our
+[Contributing Guide](https://github.com/dotnet/core-docs/blob/master/CONTRIBUTING.md)
+for details.
+
+## Issues with Existing Topics
+
+Please include a link to the topic. Describe the problem as specifically as you can. Consider:  
+
```

Comment:
I'm not sure we need to enumerate a few possible reasons. People find and issue then decide to report it. What's the benefit of providing this list?